test/success/num.prl

```diff
@@ -22,7 +22,7 @@ Thm Pred : [
   lam a.
   fresh a', ind -> elim a;
   [ `zero ];
-  [ use a' ]
+  [ `,a' ]
```

Comment:
The question is, should we strengthen the tactic `use` to accept mismatch kinds?